docs/telemetry.json

```diff
@@ -8,7 +8,7 @@
   "definitions": {
     "branch_message": {
       "type": "string",
-      "enum": ["addon_init", "download_prefs", "prompt_save_click", "prompt_cancel_click",
+      "enum": ["addon_init", "download_prefs", "prompt_opted_in", "prompt_cancel_click",
```

Comment:
This doesn't include the install/uninstall messages.

Reply:
Install and uninstall telemetry should be coming from shield utils. Updated to remove cloud_storage_install and cloud_storage_uninstall telemetry. 

Updated doc with a new telemetry  'existing_cloud_provider_download_settings', to log users that have cloud provider as their default downloads settings before opt-in from notification panel. These users are not shown notification and Move to <cloud provider> option in context menu.